valhalla/baldr/graphconstants.h

```diff
@@ -222,6 +222,7 @@ enum class Use : uint8_t {
   kCycleway = 20,          // Dedicated bicycle path
   kMountainBike = 21,      // Mountain bike trail
 
+  kLivingStreet = 23,
```

Comment:
can you make this be part of the road uses (e.g., a value of 10)? Not really a bicycle specific use. Also add to the Use strings.